app/sections/image-hotspots/index.tsx

```diff
@@ -14,31 +14,53 @@ type HotspotsProps = SectionProps & {
   heading?: string;
   description?: string;
   image: string;
+  image2: string;
   aspectRatio: "adapt" | "1/1" | "4/3" | "3/4" | "16/9";
+  duplicateImage: boolean;
 };
 
 let Hotspots = forwardRef<HTMLElement, HotspotsProps>((props, ref) => {
-  let { heading, description, image, aspectRatio, children, ...rest } = props;
+  let {
+    heading,
+    description,
+    image,
+    image2,
+    aspectRatio,
+    duplicateImage,
+    children,
+    ...rest
+  } = props;
   let imageData: Partial<WeaverseImage> =
     typeof image === "string"
       ? { url: image, altText: "Hotspots image" }
       : image;
-
+  let image2Data: Partial<WeaverseImage> =
+    typeof image2 === "string"
+      ? { url: image2, altText: "Hotspots image" }
+      : image2;
   return (
-    <Section ref={ref} {...rest} overflow="unset">
+    <Section ref={ref} {...rest} overflow="hidden">
```

Comment:
Why changing this `overflow` prop to `hidden` ? It may crop the product popup if the hotspot item is near the corner?